### PR TITLE
feat(env): add global .env

### PR DESCRIPTION
**Pull request by uze**
Added a global .env for both apps to reference from instead of having it inside of the nextjs app. 

Refer: https://github.com/t3-oss/create-t3-turbo/issues/54

### CODE REVIEW

**Comment by juliusmarminge** [comment]
Have you tested it?

**Comment by uze** [comment]
> Have you tested it?

Yep! Once you in the root of the repo you simply run `cp .env.example .env` and it'll generate your .env file then you replace the `DATABASE_URL=file:./db.sqlite` with your real URL like normal and everything works between the expo + next app. 👍🏻

**Comment by uze** [comment]
Demo: @juliusmarminge 
https://user-images.githubusercontent.com/24923657/200193642-7d8ef667-b304-4041-b4d3-63c5676bc30e.mov

File: README.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ pnpm i
 
 # Configure environment variables.
 # There is an `.env.example` in the Next.js app you can use for reference
-cp apps/nextjs/.env.example apps/nextjs/.env
```
Comment (juliusmarminge) [inline README.md:54]: Update the comment above this line

Comment (uze) [reply README.md:54]: Updated the readme @juliusmarminge

**Review by juliusmarminge — APPROVED** [review]

**Comment by fevernova90** [comment]
@uze I believe we also need the with-env inside apps/nextjs if we're going to use global .env

Screenshot 2022-12-27 at 9 13 33 AM

**Pull request merged**